server/migrations/20190325001122-rolesCol.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  up: async function (queryInterface) {
+    await queryInterface.sequelize.query(`
+        ALTER TABLE "AgentsChannels" ADD COLUMN "roles" VARCHAR(8)[];
+        UPDATE "AgentsChannels" SET "roles" = ARRAY['viewer', 'admin'];
```

Comment:
Why didn't you use the regular way of modifying tables here? Also, is this a required attribute? (And if so, make sure defaults are set, probably to viewer)

Reply:
Thanks, fixed.